roles/network_plugin/weave/tasks/seed.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Weave seed | Set seed if first time
+  set_fact:
+    seed: '{% for host in groups["k8s-cluster"] %}{{ hostvars[host]["ansible_default_ipv4"]["macaddress"] }}{% if not loop.last %},{% endif %}{% endfor %}'
```

Comment:
if it can be any mac, then go ahead and use the default one. if it must be the mac of the interface on the network, then you need to figure out which one it is first